backend/src/BuiltinCliHost/Libs/Cli.fs

```diff
@@ -101,12 +97,23 @@ let execute
         secrets = []
         dbs = Map.empty }
 
+    let types =
+      List.concat
+        [ mod'.types |> List.map PT2RT.PackageType.toRT
+          mod'.submodules.types |> List.map PT2RT.PackageType.toRT ]
+
+    let constants =
+      List.concat
+        [ mod'.constants |> List.map PT2RT.PackageConstant.toRT
+          mod'.submodules.constants |> List.map PT2RT.PackageConstant.toRT ]
+
+    let fns =
+      List.concat
+        [ mod'.fns |> List.map PT2RT.PackageFn.toRT
+          mod'.submodules.fns |> List.map PT2RT.PackageFn.toRT ]
```

Comment:
Will these still work if there are more than 1-2 layers of modules? I think there should probably be some kinda recursion here instead of this

Reply:
I Think it works, here is a test:
```
module Test =
  module Nested =
    let test1 () : Int64 =
      let _ = Builtin.printLine ("Testing Modules1")
      0L
    module Nested2 =
      let test2 () : Int64 =
        let _ = Builtin.printLine ("Testing Modules2")
        0L
      module Nested3 =
        module Nested4 =
          let test3 () : Int64 =
            let _ = Builtin.printLine ("Testing Modules3")
            0L
            
            
CliScript.Test.Nested.test1 ()
CliScript.Test.Nested.Nested2.test2 ()
CliScript.Test.Nested.Nested2.Nested3.Nested4.test3 ()
```

Result:
![Screenshot 2024-07-08 at 18 39 27](https://github.com/darklang/dark/assets/87861924/b660859f-2782-4494-b47f-7e96a874a6b2)